services/app-api/forms/defaultMeasures.ts

```diff
@@ -1417,6 +1494,13 @@ export const defaultMeasureTemplates = {
         measureDisplay: "quality",
         id: "quality-measure-table",
       },
+      {
+        type: ElementType.TextAreaField,
+        id: "additional-notes-field",
```

Comment:
should the `id: "additional-notes-field",` here also match the ones in pomMeasures.ts? The one in pomMeasures is `id: "measure-context-text",`

Reply:
the same field id might be useful